modules/ROOT/pages/logs-using.adoc

```diff
@@ -206,8 +204,8 @@ image::logs-set-surrounding.png[View Surrounding Logs]
 
 The default settings are:
 
-* Load 5 More newer logs
-* Load 5 More older logs
+* *Load 5 more* newer logs
+* *Load 5 more* older logs
 
```

Comment:
even though "newer logs" and "older logs" are also in the UI, this just seemed clearer...?